evol/step.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, name, every: int = 1, **kwargs):
         EvolutionStep.__init__(self, name, **kwargs)
         self.count = 0
         self.every = every
+        self.kwargs = kwargs
```

Comment:
This already happens in the `__init__` of `EvolutionStep`.

Reply:
d0h.